libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -404,6 +404,31 @@
         SUPPORT_HELP_CENTER_VIEWED,
         SUPPORT_IDENTITY_FORM_VIEWED,
         SUPPORT_IDENTITY_SET,
+        QUICK_START_DIALOG_VIEWED,
+        QUICK_START_DIALOG_NEGATIVE_TAPPED,
+        QUICK_START_DIALOG_POSITIVE_TAPPED,
```

Comment:
Does it make sense to rename `QUICK_START_DIALOG_VIEWED`, `QUICK_START_DIALOG_NEGATIVE_TAPPED`, and `QUICK_START_DIALOG_POSITIVE_TAPPED` to `QUICK_START_TASK_DIALOG_VIEWED`, `QUICK_START_TASK_DIALOG_NEGATIVE_TAPPED`, and `QUICK_START_TASK_DIALOG_POSITIVE_TAPPED`?  That would follow the `QUICK_START_REQUEST_DIALOG_VIEWED`, `QUICK_START_REQUEST_DIALOG_NEGATIVE_TAPPED`, `QUICK_START_REQUEST_DIALOG_NEUTRAL_TAPPED`, and `QUICK_START_REQUEST_DIALOG_POSITIVE_TAPPED` naming more closely.  I wasn't sure what the difference was until I looked at their usage in the code.

Reply:
Good idea - this will make it a bit more clear!